README.md

```diff
@@ -160,9 +196,32 @@ lets you plugin an entire sails project directly into a Trails Application.
 [`bootstrap`](https://github.com/trailsjs/trailpack-bootstrap) extends the Trails
 boot process so that a custom method can be run during application startup.
 
+
+```js
+const ExtensionTrailpack = require('trailpack/extension')
+
+module.exports = class FootprintsTrailpack extends ExtensionTrailpack {
+
+}
+```
+
 #### `misc`
 All trailpacks that don't fit previous types.
 
+
+```js
+const Trailpack = require('trailpack')
+
+module.exports = class ExampleTrailpack extends Trailpack {
+
+}
+```
+
+### Documentation
+
+- [**Trailpack Implementation Guide**](https://trailsjs.io/doc/en/ref/trailpack)
+- [**API Reference**](https://trailsjs.io/doc/en/extend/trailpack)
+
```

Comment:
It would be nice to have here a "Migration v2 to v3" that explain big changes for trails project but also for trailpacks if they need to be updated to be compatible with v3